shell/browser/webauthn/electron_authenticator_request_delegate.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "shell/browser/webauthn/electron_authenticator_request_delegate.h"
+
+namespace electron {
+// ---------------------------------------------------------------------
+// ElectronWebAuthenticationDelegate
+// ---------------------------------------------------------------------
```

Comment:
This comment is non-standard and should be removed

Reply:
Thanks for the review!
I've removed the comment